.github/workflows/ci.yml

```diff
@@ -287,28 +385,61 @@ jobs:
         run: |
           cargo test --locked --doc
-          cargo test --locked --all-targets
+          cargo test --locked
+
+      - name: Test release
+        if: |
+          matrix.kind == 'test' && matrix.profile == 'release' &&
+          !matrix.use_sysroot
+        run: cargo test --release --locked
 
-      # TODO(ry) Because CI is so slow on for OSX and Windows, we currently only run WPT on Linux.
-      - name: Configure hosts file for WPT (linux)
-        if: startsWith(matrix.os, 'ubuntu')
+      - name: Test release (in sysroot)
+        if: |
+          matrix.kind == 'test' && matrix.profile == 'release' &&
+          matrix.use_sysroot
+        run: |
+          sudo chroot /sysroot   \
+               su -l "$(whoami)" \
+                  -c "cargo test --release --locked"
+
+      # TODO(ry): Because CI is so slow on for OSX and Windows, we currently
+      #           run the Web Platform tests only on Linux.
+      - name: Configure hosts file for WPT
+        if: startsWith(matrix.os, 'ubuntu') && matrix.kind == 'test'
         run: ./wpt make-hosts-file | sudo tee -a /etc/hosts
         working-directory: test_util/wpt/
 
+      - name: Run web platform tests (debug)
+        if: |
+          startsWith(matrix.os, 'ubuntu') && matrix.kind == 'test' &&
+          matrix.profile == 'debug'
+        run: |
+          ./target/debug/deno run --allow-env --allow-net --allow-read \
+                                  --allow-run --allow-write --unstable \
+                              ./tools/wpt.ts setup
+          ./target/debug/deno run --allow-env --allow-net --allow-read \
+                                  --allow-run --allow-write --unstable \
+                              ./tools/wpt.ts run --no-build --quiet    \
+
       - name: Run web platform tests (release)
-        if: startsWith(matrix.os, 'ubuntu') && matrix.kind == 'test' && matrix.profile == 'release'
+        if: |
+          startsWith(matrix.os, 'ubuntu') && matrix.kind == 'test' &&
+          matrix.profile == 'release'
         run: |
-          deno run --unstable --allow-write --allow-read --allow-net --allow-env --allow-run ./tools/wpt.ts setup
-          deno run --unstable --allow-write --allow-read --allow-net --allow-env --allow-run ./tools/wpt.ts run --quiet --release --json=wpt.json --wptreport=wptreport.json
-          gzip ./wptreport.json
+          ./target/release/deno run --allow-env --allow-net --allow-read \
+                                    --allow-run --allow-write --unstable \
+                                ./tools/wpt.ts setup
+          ./target/release/deno run --allow-env --allow-net --allow-read     \
+                                    --allow-run --allow-write --unstable     \
+                                ./tools/wpt.ts run --no-build --quiet        \
+                                                   --release --json=wpt.json \
+                                                   --wptreport=wptreport.json
```

Comment:
So now WPT only run on release, and not on debug? If we keep going with removing WPT targets at this rate, we won't run WPT at all next month...

Reply:
No? Have you read the entire diff?

Reply:
Mh, I could have sworn I double checked... I guess not.